stonesoup/feeder/filter.py

```diff
@@ -41,3 +42,147 @@ def detections_gen(self):
                         meta_values.add(meta_value)
                 self._detections = unique_detections
             yield time, unique_detections
+
+
+class MetadataValueFilter(MetadataReducer):
+    """ Reduce detections by filtering out objects based on whether the value
+        of a particular metadata field conforms to a given condition.
+
+        The MetadataValueFilter provides an easy way of reducing detections in
+        cases where an informative metadata field exists (e.g. MMSI, SNR, etc.)
+        , that can be used to identify and filter out unwanted detections.
+
+        Once provided with a :py:attr:`~metadata_field` name and a suitable
+        :py:attr:`~operator` function, the feeder will filter the incoming
+        detections by evaluating the :py:attr:`~metadata_field` field value
+        using the desired :py:attr:`~operator` function. Only detections
+        that satisfy the operator condition (i.e. cause the
+        :py:attr:`~operator` to return `True`) are allowed through the filter.
+    """
+
+    operator = Property(
+        object,
+        doc="A unary operator/function of the form :code:`b = f(val)`, "
+            "where :code:`val` is the value of the selected `metadata_field`. "
+            "The function MUST return a :py:class:`~bool` type that evaluates "
+            "to `True` when a particul ar object satisfies the condition(s) "
+            "set by the operator, and thus should be allowed through the "
+            "filter. Any detections causing the operator to return `False` "
+            "will be filtered out. "
+            "`See here <https://docs.python.org/3/library/operator.html#module-operator>`_ "  # noqa(E504)
+            "for a list of built-in Python operators that can be readily used."
```

Comment:
I think it's worth providing an example in the main documentation section, like how to use _greater than_ (like you've used in your test).

Reply:
Done (along with other changes relating to `operator` module)